src/components/MultiBarToggle.js

```diff
@@ -231,6 +243,13 @@ const Styles = {
         flex: 1,
         alignItems: 'center',
         justifyContent: 'center'
+    },
+    overlayActive: {
```

Comment:
Why do do so much **width** and **height**?

Reply:
For some reason I wasn't able to get the full height and width of the screen.

Reply:
That's the only one I got stuck. If you have any suggestion...

Reply:
Let's try with:
`StyleSheet.absoluteFillObject`

`
{
...StyleSheet.absoluteFillObject,
backgroundColor: 'rgba(0,0,0,0.5)'
}
`

Reply:
Didn't work. 

<img src="https://user-images.githubusercontent.com/21176544/59428633-1240bf00-8dde-11e9-91c3-ef845a3cf279.png" width="200" />